src/init.jl

```diff
@@ -74,16 +76,22 @@ function __init__()
         end
     end
 
-    if haskey(ENV, "PLOTS_HOST_DEPENDENCY_LOCAL")
-        use_local_plotlyjs[] = ENV["PLOTS_HOST_DEPENDENCY_LOCAL"] == "true"
-        use_local_dependencies[] = isfile(plotly_local_file_path) && use_local_plotlyjs[]
-        if use_local_plotlyjs[] && !isfile(plotly_local_file_path)
-            @warn("PLOTS_HOST_DEPENDENCY_LOCAL is set to true, but no local plotly file found. run Pkg.build(\"Plots\") and make sure PLOTS_HOST_DEPENDENCY_LOCAL is set to true")
+    if get(ENV, "PLOTS_HOST_DEPENDENCY_LOCAL", "false") == true
```

Comment:
This path is never taken, since what you get back is a the string `"true"` if you have set it to that, so that is never `== true`

Reply:
I changed that in https://github.com/JuliaPlots/Plots.jl/pull/3067